### PR TITLE
Add Icons for configured system vendors

### PR DESCRIPTION
**Pull request by k-chu**
Add Icons for configured system vendors

### CODE REVIEW

**Comment by k-chu** [comment]
@miq-bot add_reviewer @h-kataria

**Comment by k-chu** [comment]
@miq-bot add_reviewer @h-kataria

File: app/decorators/configured_system_decorator.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@ def self.fonticon
     'ff ff-configured-system'
   end
 
+  def fileicon
+    try(:label_for_vendor_icon) ? "svg/vendor-#{label_for_vendor_icon.downcase}.svg" : nil
```
Comment (skateman) [inline app/decorators/configured_system_decorator.rb:7]: Is this method specific to a single provider? If the answer is yes, you can have a more specific decorator in the provider's repo with its quadicon defined there. This is what the AutoSDE people have done for their AutoSDE-specific [decorators](https://github.com/ManageIQ/manageiq-providers-autosde/tree/master/app/decorators/manageiq/providers/autosde) and [assets](https://github.com/ManageIQ/manageiq-providers-autosde/blob/master/app/assets/images/svg/vendor-autosde.svg).

Comment (k-chu) [reply app/decorators/configured_system_decorator.rb]: Moved the provider specific decorator logic to our provider. Thanks for the review.

**Comment by k-chu** [comment]
Moved icons and decorator code to the Provider. This is no longer needed.

**Pull request closed without merging**